.github/workflows/build.yml

```diff
@@ -20,7 +20,21 @@ jobs:
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip setuptools six wheel
-          python -m pip install pytest-cov -r requirements.txt
+          python -m pip install mypy pytest-cov -r requirements.txt
+      # FIXME: #4052 fix mypy errors in other directories and add them here
+      - run: python3 -m  mypy --ignore-missing-imports
```

Comment:
I don't think there's any need for calling `mypy` using the Python interpreter as `mypy` has its own entrypoint.